book/A-upgrading/sections/ui-theme.asc

```diff
@@ -68,8 +68,7 @@ atom-text-editor::shadow .highlight.my-linter {
 }
 ```
 
-Check out the https://github.com/atom/find-and-replace/blob/95351f261bc384960a69b66bf12eae8002da63f9/styles/find-and-replace.less#L10
-[find-and-replace]package for another example of using `::shadow` to pierce the shadow DOM.
+Check out the https://github.com/atom/find-and-replace/blob/95351f261bc384960a69b66bf12eae8002da63f9/stylesheets/find-and-replace.less#L9-L29[find-and-replace] package for another example of using `::shadow` to pierce the shadow DOM.
```

Comment:
This looks like it shouldn't be part of this PR. We can still keep it since it fixes that link.